cli

```diff
@@ -84,37 +62,11 @@ if (program.datadir) {
     process.env.DATADIR = program.datadir;
 }
 
-function getContractAddress(name) {
-    const filename = path.resolve(__dirname, 'truffle', 'build', 'contracts', `${name}.json`);
-
-    if (!existsSync(filename)) {
-        return;
-    }
-
-    const {networks} = require(filename);
-
-    for (const network in networks) {
-        const {address} = networks[network];
-        if (address && typeof address === 'string') {
-            return address;
-        }
-    }
-}
-
-// ------------------ Patch Config ------------ //
-
-if (process.env.MODE === 'e2e') {
-    process.env.IDENTITY_CONTRACT_ADDRESS = getContractAddress('Identity');
-    process.env.STORAGE_PROVIDER_REGISTRY_CONTRACT_ADDRESS = getContractAddress('StorageProviderRegistry');
-}
-
 const config = require('config');
-const logger = require('./core/log');
-const Model = require('./db/model');
-const Deployer = require('./client/zweb/deployer');
-const DB = require('./db');
-
-ctx.basepath = __dirname;
+const logger = require('./src/core/log');
+const Console = require('./src/console');
+const Deployer = require('./src/client/zweb/deployer');
+const DB = require('./src/db');
```

Comment:
Looks like we don't need `logger`, `Deployer` and `DB` imports here. Also I am not sure the `Console` import will work when we rename the file to `ts` in the `src` directory.

Reply:
We do as we have logger below (well, we can replace it with console). Deployer and DB are needed for deploy and migrate scripts.
I didn't change this, the file was just split